admin/mkassoc

```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/env ruby
 
 require "openid/consumer/associationmanager"
-require "openid/store/memstore"
+require "openid/store/memory"
 
-store = OpenID::MemoryStore.new
+store = OpenID::Store::Memory.new
 ARGV.each do |server_url|
-  mgr = OpenID::Consumer::AssociationManager.new(store, URI.parse(server_url))
+  mgr = OpenID::Consumer::AssociationManager.new(store, server_url)
```

Comment:
Why have you removed the `URI.parse` call?
I would guess that it's there to ensure that the arguments are valid URIs?